tad/migrations/versions/b62dbd9468e4_create_status_user_and_task_table.py

```diff
@@ -21,7 +21,7 @@
 
 def upgrade() -> None:
     # ### commands auto generated by Alembic - please adjust! ###
-    status = op.create_table(
+    op.create_table(
```

Comment:
you should not edit migrations. You will need to create a new migration that undoes the current migration

Reply:
My bad. I will create a new migration.